app/decorators/application_choice_export_decorator.rb

```diff
@@ -1,10 +1,18 @@
 class ApplicationChoiceExportDecorator < SimpleDelegator
+  RELEVANT_SUBJECTS = [
```

Comment:
Is this equivalent to `ApplicationQualification::REQUIRED_GCSE_SUBJECTS`?

Reply:
@duncanjbrown amended here https://github.com/DFE-Digital/apply-for-teacher-training/commit/dca2b50620ccf2733a192213748b057621b6b9d1

Reply:
👍